apps/codeforafrica/src/lib/data/pagify/index.js

```diff
@@ -1,18 +1,20 @@
-import members from "./members";
-import partners from "./partners";
+import member from "./member";
+import partner from "./partner";
 import post from "./post";
+import project from "./project";
 
 const COLLECTION_BY_SLUG = {
-  members,
+  members: member,
```

Comment:
Again, not sure why we're mixing plural and singular here. What's the real issue?

Reply:
given path = `/about/members/:slug`,
We need to get a single member, so we have collection slug as members.

Reply:
Looks like we're going to have to review this part when we fix the 3 item pagination functionality...